src/pages/index.js

```diff
@@ -12,12 +12,12 @@ import contactIcon from '../assets/images/contact-icon.png';
 const IndexPage = ({ data }) => (
   <div className={styles.pageContainer}>
     <div className={styles.intro}>
-      <h1>{data.site.siteMetadata.title}</h1>
-      <p className={styles.tagline}>It's not as harmless as you think.</p>
+      <h1 className={styles.pageTitle}>{data.site.siteMetadata.title}</h1>
+      <p className={styles.tagline}>Its not as harmless as you think.</p>
```

Comment:
Should it be `its` or `it's` here? I think `it's` is short for "it is" which makes sense. I think without the apostrophe "its" is possessive!

Reply:
Ah! Thanks! I forgot to switch it back. Atom does this thing where it highlights everything after the apostrophe!

![screen shot 2018-09-14 at 3 11 23 pm](https://user-images.githubusercontent.com/37306074/45577399-a4f10500-b830-11e8-807e-e0680df40ec2.png)